tools/tertiary-analysis/pyscenic/pyscenic_aucell.xml

```diff
@@ -0,0 +1,110 @@
+<tool id="pyscenic_aucell" name="PySCENIC AUCell" profile="21.09" version="0.12.1+galaxy0">
```

Comment:
increase the profile version?

Reply:
This is the latest I can use unfortunately before the next one (23.0) that will work with our instances (22.x).